src/Command/SendTeachingRemindersCommand.php

```diff
@@ -215,16 +233,15 @@ protected function execute(InputInterface $input, OutputInterface $output)
      */
     protected function getTemplatePath(SchoolInterface $school)
     {
-        $paths = [
```

Comment:
see comment above about loss of overriding ability of instance wide template. 
on a related note, this method is the same as in the other mail command - suggesting refactoring this out into a mixin or base class?

Reply:
It's the same, I'd worry about a trait though because it relies on the constant that is set here. I'm not sure how I feel about that as the trait would be invalid without the overridden constant. Base class has the same issue without adding an interface and a method to get the template name.

Reply:
yea, merely a thought. this is acceptable.